plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml

```diff
@@ -128,6 +128,7 @@ items:
     - ""
     resources:
     - configmaps
+    - endpoints
```

Comment:
maybe we can restrict only to "updates", I think that we only require access to add and remove annotations ...

Reply:
or adding it separately as suggested in the docs https://kubernetes.io/docs/reference/access-authn-authz/rbac/#write-access-for-endpoints ?
at least a reference to know why we bring it back, or this can pop up on some security static checkers

Reply:
I see, this is the reverted commit https://github.com/openshift/kubernetes/commit/416efdab26afe06cf2b57991dfac511769bf508b

Reply:
+1 separating the rule out and making it as small as possible to enable `oc idle`. The change as it is will create rebase problems in the future.